packages/flutter/test/cupertino/theme_test.dart

```diff
@@ -206,6 +206,41 @@ void main() {
     );
   });
 
+  test('Theme operator == returns correct value for simple cases', () {
+    expect(const CupertinoThemeData(), equals(const CupertinoThemeData()));
```

Comment:
nit (and unfortunate that our automatic formatter doesn't do it for you): but for consistent styling, please keep the line formatting consistent with the rest of the file. Mainly that closing parenths/brackets etc are in matching indents as the opening parenths/brackets. You can trigger it in the formatter by using trailing commas